playwright/.env.playwright

```diff
@@ -0,0 +1,6 @@
+PORT=3000
+CLIENT_JIRA_CLIENT_ID=32uKfFc4hS5fjP7SZqVXKjC65IBveAGX
```

Comment:
For security reasons, we probably should not be tracking the client ID & Secret in the .env file.

They should be added manually by the developer when running the end to end tests locally. 
And when its being ran in the pipeline, it should come from github secrets or something.

What do you think?

Reply:
@shoumaw, it makes sense, but keep in mind that these are the client ID and secret for the generic test account testuser@bitovi.com. The main .env file for each developer account is not(and should not) be committed.